crates/bevy_render/src/camera/bundle.rs

```diff
@@ -75,6 +75,23 @@ pub struct OrthographicCameraBundle {
 }
 
 impl OrthographicCameraBundle {
+    /// Create an orthographic projection camera to render 2D content.
+    ///
+    /// The projection creates a camera space where X points to the right of the screen,
+    /// Y points to the top of the screen, and Z points out of the screen (backward),
+    /// forming a right-handed coordinate system. The center of the screen is at `X=0` and
+    /// `Y=0`.
+    ///
+    /// The default scaling mode is [`ScalingMode::WindowSize`], resulting in a resolution
+    /// where 1 unit in X and Y in camera space corresponds to 1 logical pixel on the screen.
+    /// That is, for a screen of 1920 pixels in width, the X coordinates visible on screen go
+    /// from `X=-960` to `X=+960` in world space, left to right. This can be changed by changing
+    /// the [`OrthographicProjection::scaling_mode`] field.
+    ///
+    /// The camera is placed at `Z=+1000-0.1`, looking toward the world origin `(0,0,0)`.
+    /// Its orthographic projection extends from `0.0` to `1000.0` in camera view space,
```

Comment:
0.0 to -1000.0 in camera view space as the projection has `near` = 0.0 and `far` = 1000.0 and is right-handed y-up. `near` and `far` refer to parameters of the projection as to the distance to the closest and farthest objects that will be rendered by the projection. Right-handed y-up still means that positive z points out of the screen, so 1000.0 units in front of the camera is at z = -1000.0 in camera view space. Hopefully that was a clear elaboration but let me know if something was confusing or incorrect. :)

Reply:
Right, "Z is inverted".

Reply:
> the distance to the closest and farthest objects

Note that I explicitly avoided that phrasing because it's not correct; clipping occurs at the fragment level in shaders, not object level. There may be CPU-side clipping too at object level, but technically an object at distance Z=-999.99 may be partially invisible if it extends beyond 1000.0 units from the camera, so it's not a boolean show/hide at the object level.

Reply:
Fixed.